src/fidesops/schemas/saas/saas_config.py

```diff
@@ -73,11 +76,18 @@ class SaaSRequest(BaseModel):
     """
 
     path: str
+    method: Optional[HTTPMethod]
```

Comment:
As mentioned, this isn't an optional field in a HTTP request and introduces a potentially confusing default setting problem.